ui/src/app/archived-workflows/components/archived-workflow-details/archived-workflow-details.tsx

```diff
@@ -273,6 +273,12 @@ export class ArchivedWorkflowDetails extends BasePage<RouteComponentProps<any>,
                 finishedAt: this.state.workflow.status.finishedAt
             }
         };
-        document.location.href = ProcessURL(link.url, object);
+        const url = ProcessURL(link.url, object);
+
+        if ((window.event as MouseEvent).ctrlKey || (window.event as MouseEvent).metaKey) {
+            window.open(url, '_blank');
+        } else {
+            document.location.href = url;
+        }
```

Comment:
Looks like archived workflow links don't support opening with a new tab before. So fixed that.